tests/test_crypto.py

```
@@ -0,0 +1,63 @@
+def test_address_derivation_is_deterministic() -> None:
+    signing_key, verify_key = generate_key_pair()
+    first = derive_address(verify_key)
+    second = derive_address(verify_key)
+
+    assert first == second
+    assert first == derive_address(signing_key.verify_key)
+    assert len(first) == 40
+
+
+def test_invalid_signature_is_rejected() -> None:
+    signing_key, verify_key = generate_key_pair()
```

Ruff (0.15.1)

[warning] Unpacked variable `signing_key` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)